core/src/App.html

```diff
@@ -966,6 +974,28 @@
     }
   };
 
+  //Dialog Box
+  userSettingsInternalObj = {};
+  export const openUserSettings = () => {
+    const userSettingsSchema = LuigiConfig.getConfigValue('settings.userSettings.userSettingGroups');
+    if (GenericHelpers.isObject(userSettingsSchema)) {
+      const userSettingsData = [];
+      for (const item in userSettingsSchema) {
+        let innerObj = {};
+        innerObj[item] = userSettingsSchema[item];
+        userSettingsData.push(innerObj);
+      };
+      userSettingsInternalObj.userSettingsData = userSettingsData;
+      userSettingsInternalObj.displayed = true;
+    } else {
+      console.info('There is no user settings schema defined.');
+    }
```

Comment:
Would it be possible to move this logic to a new Helper like /utilities/dialog-helper.js....
For two main reasons:
1)App/html feature after feature, it is increasing... and I am getting older and it is becoming always harder to manage big files.... 
2)Most important, it is easier to write unit tests on external js....

In this case we have few lines, but there are other function like EventListenerHelpers.addEventListener('message',  where we have 100 of lines....

Reply:
👍 Good point(s). Thx, I will move it